tests/e2e/docker/hermes.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM informalsystems/hermes:0.12.0 AS hermes-builder
```

Comment:
guess it would make sense to pass the hermies version via a config/env just so that we can use different published images. wdyt?